webapp/apps/btax/views.py

```diff
@@ -323,10 +322,9 @@ def output_detail(request, pk):
 
     model = url.unique_inputs
     if model.tax_result:
-        output = url.unique_inputs.tax_result
+        tables = url.unique_inputs.tax_result[0]
```

Comment:
The result returned by dropq (`tax_result`) is always a list, I think as long as the number of separate jobs, 1 in this case.